### ISSUE TITLE
GT06N LBS Information Packet 0x11 decode error

### ISSUE THREAD

**Issue by CyberPK** [traccar/traccar#4509]
raw data example: 7878131114040a0d1f22xxxxxxxxxxxxxxxx00b806ef0d0a
decoded as
{"mcc":5124,"mnc":10,"lac":3359,"cid":xxx,"raw":"xxxxxxxxx00b806ef0d0a","distance":0.0,"totalDistance":xxxx,"ip":"xxxxx","motion":true}{"radioType":"gsm","considerIp":false,"cellTowers":[{"cellId":xxxxx,"locationAreaCode":3359,"mobileCountryCode":5124,"mobileNetworkCode":10}]}

but analyzing:

7878 Start Bit
13   Packet Length. Length = Protocol Number + Information Content + Information Serial Number + Error Check, totally (5+N)Bytes, because the Information Content is a variable length field.
----
11   LBS Information Packet

14  year
04  month
0a  day
0d  hour
1f  minute
22  seconds

xxxx mcc
xx    mnc
xxxx lac
xxxxxx cellid
00b8 information SN
06ef Error Check
---
0d0a Stop Bit

I've looked at the code and found that the LBS packages parser miss the datetime, consuming the datetime bytes as lbs data.
Every GT06N communicatin protocol document I've found has the datetime, so I think it is not a problem of my device.

Thank you

**Comment by tananaev** [comment]
Do you have documentation for your device?

**Comment by CyberPK** [comment]
No, as I own only 1 device, but I've found some documentation (linked below) that adhere to the raw data.
Cannot find any other protocol documentation for GT06N without datetime on LBS

[https://filebin.net/ufd7a1i2ifum7ylj](url)

**Comment by tananaev** [comment]
Looks like you are right. It's not decoded correctly.

**Comment by CyberPK** [comment]
Thank you

**Comment by CyberPK** [comment]
I've created the patch by myself.

[traccar patch fix lbs gt06.patch.txt](https://github.com/traccar/traccar/files/4465127/traccar.patch.fix.lbs.gt06.patch.txt)

**Comment by tananaev** [comment]
Feel free to send a pull request.

**Comment by CyberPK** [comment]
Sorry, there is an error in case arrives a combined packet.
I'll submit a new patch to fix it

**Comment by CyberPK** [comment]
this patch closes #4509 #4510 #4511
[patch.diff.txt](https://github.com/traccar/traccar/files/4470138/patch.diff.txt)
